packages/frontend/src/helpers/flow-templates.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from 'react'
+import {
+  BiBell,
+  BiCalendar,
+  BiCheckDouble,
+  BiEnvelope,
+  BiFile,
+  BiInfoCircle,
+  BiListUl,
+  BiMessageAlt,
+  BiQuestionMark,
+  BiStar,
+} from 'react-icons/bi'
+
+import type {
+  CreateTemplatedFlowInput,
+  DemoVideoDetails,
+} from '@/graphql/__generated__/graphql'
+
+// Remember to sync with the backend!
+const SEND_FOLLOW_UPS_NAME = 'Send follow ups'
+const SEND_A_COPY_OF_FORM_RESPONSE_NAME = 'Send a copy of form response'
+const SCHEDULE_REMINDERS_NAME = 'Schedule reminders'
+const TRACK_FEEDBACK_NAME = 'Track feedback'
+const ATTENDANCE_TAKING_NAME = 'Attendance taking'
+const UPDATE_MAILING_LISTS_NAME = 'Update mailing lists'
+const ROUTE_SUPPORT_ENQUIRIES_NAME = 'Route support enquiries'
+const GET_LIVE_UPDATES_THROUGH_TELEGRAM_NAME =
+  'Get live updates through Telegram'
+const SEND_MESSAGE_TO_A_SLACK_CHANNEL_NAME = 'Send message to a Slack channel'
```

Comment:
The template name shouldn't be the template identifier? but either ways, we shouldn't need this here.